open-api/rest-catalog-open-api.py

```diff
@@ -441,6 +441,30 @@ class AssertViewUUID(BaseModel):
     uuid: str
 
 
+class AzureCredentials(BaseModel):
+    account_name: Optional[str] = Field(None, alias='account-name')
+    account_key: Optional[str] = Field(None, alias='account-key')
+    token: Optional[str] = None
+
+
+class AwsCredentials(BaseModel):
+    access_key_id: str = Field(..., alias='access-key-id')
+    secret_access_key: str = Field(..., alias='secret-access-key')
+    token: Optional[str] = None
+    expires_at_ms: Optional[int] = Field(None, alias='expires-at-ms')
+
+
+class GcsCredentials(BaseModel):
+    token: str
+    expires_at_ms: Optional[int] = Field(None, alias='expires-at-ms')
+
```

Comment:
```suggestion
    default_bucket_location: Optional[str] = Field(None, alias='default-bucket-location')
    endpoint: Optional[str] = None
```

Reply:
I don't think we want to include those, because we only want to standardize the credentials themselves and bucket / endpoint isn't credential-related